src/geom_handlers/common_assessment.jl

```diff
@@ -231,5 +230,39 @@ function filter_intersecting_sites(res_df::DataFrame)::DataFrame
         end
     end
 
-    return res_df[res_df.row_ID .∉ [unique(ignore_list)], :]
+    rename!(res_df, :poly => :geometry)
+
+    return res_df[res_df.row_ID .∉ [unique(ignore_list)], Not(:qc_flag, :row_ID)]
+end
+
+"""
+    output_geojson(
+        df::DataFrame,
+        region::String,
+        output_dir::String
+    )::Nothing
+
+Writes out GeoJSON file to a target directory. Output file will be located at location:
+"`output_dir`/output_dir_`region`_`current_date_time`.geojson"
+
+# Arguments
+- `df` : DataFrame intended for writing to geojson file.
+- `region` : Region name for labelling output file.
+- `output_dir` : Directory to write geojson file to.
+"""
+function output_geojson(
+    df::DataFrame,
+    region::String,
+    output_dir::String
+)::Nothing
+    GDF.write(
+        joinpath(
+            output_dir,
+            "output_sites_$(region)_$(Dates.format(now(), "Y-mm-dd_THH-MM")).geojson"
```

Comment:
Is this date format expected?

Also, do we need a date/time specific id in the filename?

In the preprocessing, it's handy as it's immediately clear when the data was preprocessed. In this use case, I'm not sure of the reason/benefit.

Reply:
I wasn't sure how the API/web app retrieves the output geojson files but if the date and time information isn't useful there then I'll remove it.